packages/keycodes/src/index.js

```diff
@@ -24,77 +24,103 @@ import { __ } from '@wordpress/i18n';
  */
 import { isAppleOS } from './platform';
 
-/**
- * @typedef {'primary'|'primaryShift'|'primaryAlt'|'secondary'|'access'|'ctrl'|'alt'|'ctrlShift'|'shift'|'shiftAlt'} WPKeycodeModifier
- */
+/** @typedef {typeof ALT | CTRL | COMMAND | SHIFT } WPModifierPart */
```

Comment:
Another disadvantage to JSDoc is that these types will be public. They cannot be declared as typedefs here without being exported.